### PR TITLE
A11y - Remove alt text from podcasts

### LINKED ISSUE #8910: Podcast Header Images - Use Empty Alt Text
**Is your feature request related to a problem? Please describe.**
The alt text for the image in podcast pages is the default alt text for the service

We've been asked to remove this for a11y UX, as the current string is unhelpful

**Describe the solution you'd like**
Set a blank alt text for that image, on podcasts only.  Radio episodes should still have the existing implementation

**Context/discussion**
https://bbcnews.slack.com/archives/CK85Q5SL9/p1614005259001200

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by ryanmccombe**
Resolves https://github.com/bbc/simorgh/issues/8910

**Overall change:**
_Removes unnecessary alt text from podcast images._

**Code changes:**

- Remove unnecessary alt text from podcast images.

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/routes/onDemandAudio/getInitialData/index.js [hunk 1]
```diff
@@ -134,6 +134,8 @@ export default async ({ path: pathname, pageType, service, toggles }) => {
           ? getPodcastPageIdentifier(pageIdentifier)
           : pageIdentifier,
         imageUrl: get(['content', 'blocks', 0, 'imageUrl'], LOG_LEVELS.INFO),
+        // An empty string in alt text will cause it not to be read out by AT; a null value will fall back to the default
```
Comment (ryanmccombe) [inline src/app/routes/onDemandAudio/getInitialData/index.js]:
```suggestion
        // An empty string in alt text will not be read out by AT; a null value will fall back to the default
```

File: src/app/routes/onDemandAudio/getInitialData/index.js [hunk 2]
```diff
@@ -134,6 +134,8 @@ export default async ({ path: pathname, pageType, service, toggles }) => {
           ? getPodcastPageIdentifier(pageIdentifier)
           : pageIdentifier,
         imageUrl: get(['content', 'blocks', 0, 'imageUrl'], LOG_LEVELS.INFO),
+        // An empty string in alt text will not be read out by AT; a null value will fall back to the default
```
Comment (greenc05) [inline src/app/routes/onDemandAudio/getInitialData/index.js]: How about "An empty string/null value for alt text results in the image not being acknowledged by AT;"

What is the default referring to @ryanmccombe?

**Review by pharingee — APPROVED** [review]

File: src/app/routes/onDemandAudio/getInitialData/index.js (hunk 2 above)
Comment (pharingee) [reply src/app/routes/onDemandAudio/getInitialData/index.js]: @greenc05 we have default service level alt text for all images on Simorgh. So when the value is null, that default will be used. But when the value is an empty string i.e. `''` then the default alt text that simorgh uses will not be applied. And that's what the rest of this PR ensures.

E.g. The default alt image for bbc.com/afrique is 'BBC News Afrique'. If an image on simorgh doesn't come with alt text (i.e. `''` or `null`), we usually apply this to it. This is true for the podcast and on demand radio image as well. This PR differentiates between `null` and `''` by using the empty string `''` when it's given and using 'BBC News Afrique' when it's null. That way we can change the behaviour for podcast images without changing the behaviour for the rest of the site

Comment (greenc05) [reply src/app/routes/onDemandAudio/getInitialData/index.js]: Thanks for the detailed explanation @pharingee

Comment (ryanmccombe) [inline src/app/routes/onDemandAudio/getInitialData/index.js]:
```suggestion
        // An empty string in alt text will not be read out by AT; a null value will use the default alt text from the service config
```

**Review by rebeccamcginn — APPROVED** [review]

**Pull request merged**